hl7/4.2/modules/ROOT/edits/hl7-connector-studio.adoc

```diff
@@ -0,0 +1,99 @@
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure HTTP Listener as the source for the HL7 EDI Connector to initiate a flow each time the listener receives a request on the configured host and port.
+
+To configure an HTTP Listener source, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP > Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the HTTP Listener source in the app.
+. On the *General* tab, specify connection information for the connector.
+. On the *TLS* tab, optionally specify the TLS information for the connector.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you are specifying an action for that connector to perform.
+
+To add an operation for HL7 EDI Connector, follow these steps:
+
+. In the *Mule Palette* view, select *HL7* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target HL7 EDI system.
+
+To configure the global element for HL7 EDI Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
++
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties.
+
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[configure-other-fields]]
+== Configure Additional Connector Fields
+
+After you configure a global element for <connector name>, configure the the other required fields for the connector. The required fields vary depending on which connector operation you use.
```

Comment:
Replace the boilerplate text here with the actual name.

Reply:
Fixed.